source/blog/2017-04-27-rspec-3-6-has-been-released.md

```diff
@@ -0,0 +1,94 @@
+---
+title: RSpec 3.6 has been released!
+author: Sam Phippen, TODO: other editors
+---
+
+RSpec 3.6 has just been released! Given our commitment to
+[semantic versioning](http://semver.org/), this should be a trivial
+upgrade for anyone already using RSpec 3, but if we did introduce
+any regressions, please let us know, and we'll get a patch release
+out with a fix ASAP.
+
+RSpec continues to be a community-driven project with contributors
+from all over the world. This release includes over XXX commits and YYY
+merged pull requests from over 50 different contributors!
+
+Thank you to everyone who helped make this release happen!
+
+## Notable Changes
+
+### Core: Failure counts of errors occurring outside examples
+
+In previous versions of RSpec, when errors occurred outside of examples, they
+could potentially have side effects like preventing examples from running, or
+stopping nested example groups from being defined. To help debug those issues,
+RSpec now prints when those errors occur in the end of run summary, for
+example:
+
+~~~
+Finished in 0.00033 seconds (files took 0.14097 seconds to load)
+0 examples, 0 failures, 1 error occurred outside of examples
+~~~
+
+Special thanks to Jon Rowe for adding this feature.
+
+### Core: `config.fail_if_no_examples`
+
+As it currently stands RSpec will exit with code `0` indicating success if no
+examples are defined. This configuration option makes it possible to cause RSpec
+to exit with code `1` indicating failure, which is useful for CI environments.
```

Comment:
I think it would be better to more explicitly explain why this is useful--i.e. it helps detect when you've misconfigured RSpec to look for specs in the wrong place or with the wrong pattern and it therefore can't find any.  Instead of always passing, this makes it fail in that case.